Apps/package-lock.json

```diff
@@ -1,1854 +1,1854 @@
-{

```

Comment:
Why is this file changed?

Reply:
Looks like it was a change included when I merged master in yesterday.  Lemme see if I can fix it

Reply:
Looks like it was a change included when I merged master in yesterday.  Lemme see if I can fix it